plugins/brimcap/brimcap-plugin.ts

```diff
@@ -171,27 +169,24 @@ export default class BrimcapPlugin {
     )
   }
 
-  private logToSearchOpts(log: zng.Record): searchOptions {
-    const ts = log.get("ts") as zng.Primitive
+  private logToSearchOpts(log: zed.Record): searchOptions {
+    const ts = log.get("ts") as zed.Time
     // RFC3999nano format with zero timezone offset
-    const formatter = DateTimeFormatter.ofPattern(
-      "yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSS'Z'"
+    // const formatter = DateTimeFormatter.ofPattern(
+    //   "yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSS'Z'"
+    // )
+
+    const tsString = ts.toString()
+    const dur = log.get("duration") as zed.Duration
+    const dest = join(
+      this.api.getTempDir(),
+      `packets-${ts.toString()}.pcap`.replaceAll(":", "_")
```

Comment:
Is this the fix for Windows you mentioned?

Reply:
It is!